lib/controllers/__tests__/messages.test.ts

```diff
@@ -56,6 +56,18 @@ import {
   MessagePayloadMiddleware
 } from "../messages";
 
+import * as lolex from "lolex";
+
+// mock time (ie. created_at)
+// tslint:disable-next-line:no-let
+let clock: any;
```

Comment:
not used yet?

Reply:
what exactly ?

Reply:
that variable

Reply:
yes that's because it must be a global one to be shared by afterEach and beforeEach methods